dist/ansible/scripts/ovn-setup.sh

```diff
@@ -10,23 +10,84 @@
 # This script is run as part of installation after the cluster is
 # up and before the ovn daemonsets are created.
 
-health=$(kubectl get --raw /healthz/ready)
-if [ $health != 'ok' ]
-then
-  echo " cluster not ready"
-  exit 1
-fi
+trap 'kill $(jobs -p); exit 0' TERM
+
+cfg=${1:-"/etc/origin/master/master-config.yaml"}
+
+wait_for_cluster_up () {
```

Comment:
These two wait functions basically do the same thing except for the specific command they run, perhaps we should just steal wait-for-condition() from OpenShift's images/dind/node/openshift-dind-lib.sh?

Reply:
@dcbw, I will take a look.